api/DOMTokenList.json

```diff
@@ -196,13 +196,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMTokenList/entries",
           "support": {
             "chrome": {
-              "version_added": "45"
+              "version_added": "42"
```

Comment:
```suggestion
              "version_added": "45"
```

45 was correct, see https://storage.googleapis.com/chromium-find-releases-static/734.html#734997b4ae07d305c47087ada6754a0bf8597a9e and follow links to https://chromium.googlesource.com/chromium/src/+/734997b4ae07d305c47087ada6754a0bf8597a9e%5E%21/#F4 to see what was introduced in M45.

Can you file a bug for what went wrong with the pipeline here?

Reply:
That doesn't make much sense...  Checking the prototype for `entries` on Chrome 42 shows it was implemented:
<img width="461" alt="Screen Shot 2020-08-28 at 10 06 00" src="https://user-images.githubusercontent.com/5179191/91594035-13dca100-e916-11ea-86ba-cbc101ef2929.png">

Reply:
This is strange. I've confirmed M42 with http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8402. Then I used bisect-builds.py to narrow it down to this range:
https://chromium.googlesource.com/chromium/src/+log/a8a9e2cd46440abd315cc876a8f6740ded878a0b..e75311d05a5839a37e20dbf8998080f2db9627ab

I think the issue is probably that this is before Blink was merged into Chromium.

Reply:
And the commit we had found was in that Blink roll:
https://chromium.googlesource.com/chromium/blink/+log/0a1a0a2..a109415

So 42 is correct. I'm not sure why the omaha proxy tools says the commit was in 45, but in any case, next time I'll need to be extra careful with Chromium changes before https://chromium.googlesource.com/chromium/src/+/b59b6df51a249895fbba24f92b661f744e031546

Reply:
The Blink merge (commit b59b6df51a249895fbba24f92b661f744e031546) was in Chromium M47 so the general rule of thumb here would be to be extra skeptical when Chromium source code archeology suggesting 47 or earlier. The Blink commits at the time were in another repository) and the mapping to Chromium milestones is even trickier.